sklearn/linear_model/tests/test_base.py

```diff
@@ -233,9 +238,9 @@ def test_linear_regression_sparse_equal_dense(normalize, fit_intercept):
     assert_allclose(clf_dense.coef_, clf_sparse.coef_)
 
 
-def test_linear_regression_multiple_outcome(random_state=0):
+def test_linear_regression_multiple_outcome(global_random_seed):
     # Test multiple-outcome linear regressions
-    X, y = make_regression(random_state=random_state)
+    X, y = make_regression(random_state=global_random_seed)
```

Comment:
Passes with all seeds. In this case I am not sure if checking for 100 different random data regression cases makes sense. Do we want that?